specificatie/openapi.yaml

```diff
@@ -169,6 +167,7 @@ paths:
           schema:
             type: integer
             maximum: 99999
```

Comment:
niet gelijk aan de definitie in de request body. Hier is het een getal tussen 1 en 99999. In de request body is het een getal tussen 0 en 99999

Reply:
Aangepast